modules/JsonExporter/src/main/java/uk/ac/ox/oii/jsonexporter/JSONExporter.java

```diff
@@ -0,0 +1,277 @@
+
+                    GraphNode jNode = new GraphNode(id);
+                    jNode.setLabel(label);
+                    jNode.setX(x);
+                    jNode.setY(y);
+                    jNode.setSize(size);
+                    jNode.setColor(color);
+
+                    
+                    for (Column col : attModel) {
+                            String name = col.getTitle();
+                            if (name.equalsIgnoreCase("Id") || name.equalsIgnoreCase("Label")
+                                    || name.equalsIgnoreCase("uid")) {
+                                continue;
+                            }
+
+                            Object valObj = n.getAttribute(col);
+                            if (valObj == null) {
+                                continue;
+                            }
+                            String val = valObj.toString();
+                            jNode.putAttribute(name, val);
+
+                    }
+
+                    jNodes.add(jNode);
+
+                    if (cancel) {
+                        return false;
+                    }
+                    Progress.progress(progress);
+                }
+
+
+                //Export edges. Progress is incremented at each step.
+                HashSet<GraphElement> jEdges = new HashSet<GraphElement>();
+                Edge[] edgeArray = graph.getEdges().toArray();
+
+                for (Edge e : edgeArray) {
+                    String sourceId = e.getSource().getId().toString();
+                    String targetId = e.getTarget().getId().toString();
+
+                    /*if (renumber) {
+                     sourceId = nodeIdMap.get(sourceId);
+                     targetId = nodeIdMap.get(targetId);
+                     }*/
+
+
+                    GraphEdge jEdge = new GraphEdge(String.valueOf(e.getId()));
+                    jEdge.setSource(sourceId);
+                    jEdge.setTarget(targetId);
+                    jEdge.setSize(e.getWeight());
+
+                    boolean mixColors=false;
+                    String color="";
+                    jEdge.setLabel(e.getLabel());
+
+                    float r=e.r();
+                    float g=e.g();
+                    float b=e.b();
+
+                    if (r==-1 || g==-1 || b==-1) {
+                        //Mix colors
+                        mixColors=true;
+                    } else {
+                        color = "rgb(" + (int) (r* 255) + "," + (int) (g* 255) + "," + (int) (b* 255) + ")";
+                    }
+
+                    Iterator<Column> eAttr = e.getAttributeColumns().iterator();
+                    while (eAttr.hasNext()) {
+                        Column col = eAttr.next();
+                        if (col == null) {
+                            continue;
+                        }
+                        String name = col.getTitle();
+                        if (name.equalsIgnoreCase("Id") || name.equalsIgnoreCase("Label")
```

Comment:
Another way to exclude all property columns is to do `!col.isProperty()`. That would exclude id, label, weight (for edge)